app/src/main/java/uw/hcrlab/kubi/lesson/HintArrayAdapter.java

```diff
@@ -0,0 +1,41 @@
+package uw.hcrlab.kubi.lesson;
+
+import android.content.Context;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.ArrayAdapter;
+import android.widget.TextView;
+
+import java.util.ArrayList;
+
+import uw.hcrlab.kubi.R;
+
+/**
+ * Created by Alexander on 4/28/2016.
+ */
+public class HintArrayAdapter extends ArrayAdapter<PromptData.Hint> {
+    private final Context context;
+    private final ArrayList<PromptData.Hint> hints;
+
+    public HintArrayAdapter(Context context, ArrayList<PromptData.Hint> hints) {
```

Comment:
Ok. I'll remove the HintCollection then and replace it with a simple ArrayList of hints.